CHANGELOG.rst

```diff
@@ -1,6 +1,77 @@
+- Simplified Python model customization. Custom preprocessing and postprocessing logic, as well as data dependencies, can be easily included in models with the python_function flavor using updated mlflow.pyfunc Python APIs. For more information, see the `Custom Python Models documentation <https://mlflow.org/docs/latest/models.html#custom-python-models>`_. (#791, #792, #793, #830, #910, @dbczumar)
+- Plugin systems allowing third party libraries to extend MLflow functionality. The `proposal document <https://gist.github.com/zblz/9e337a55a7ba73314890be68370fa69a>`_ gives the full detail of the three main changes: 
+
+  - Additional providers of tracking stores can be registered using the ``mlflow.tracking_store`` entrypoint. (#881, @zblz)
+  - Additional providers of artifact repository can be registered using the ``mlflow.artifact_repository`` entrypoint. (#882, @mociarain)
+  - The logic generating run metadata from the run context (e.g. ``source_name``, ``source_version``) has been refactored into an extendable system of run context providers. Plugins can register additional providers using the `mlflow.run_context_provider` entrypoint, which add to or overwrite tags set by the base library. (#913, #926, #930, #978, @acroz)
+
+- Support for authentication to the tracking server in the R client. Now R users can connect to secure tracking servers via credentials set in environment variables, or provide custom plugins for setting the credentials. As an example, in this release is the Databricks plugin which can detect existing Databricks credentials to allow users to connect to the Databricks tracking server. (#938, #959, #992, @tomasatdatabricks)
+
+
+Breaking changes:
+
+- [Scoring] Use pandas-split format as default for JSON requests to pyfunc scoring server (#960, @dbczumar)
+- [Scoring] When reading pandas dataframes from json, do not infer data types (#916, @mparkhe)
+
+
+More features and improvements:
+
+- [UI] Add a button for downloading artifacts (#967, @mateiz)
+- [CLI] Add ``mlflow runs`` CLI utility (#720, @DorIndivo)
+- [CLI] ``MLFLOW_EXPERIMENT_NAME`` env variable and ``--experiment-name`` arguments for CLI command (#889, #894, @mparke)
+- [Examples] Add Image classification example with Keras. (#743, @tomasatdatabricks )
+- [R][Runs] Allow client to infer context info when creating new run in fluent API (#958, @tomasatdatabricks)
+- [Logging] LogBatch support (#950, #951, #955, @smurching)
```

Comment:
Needs a bit more detail.. is there a link to the docs for example?

Reply:
This one is not done I think - maybe we shouldn't mention it in that case? @smurching